server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSOptionsCheckBaseUtility.java

```diff
@@ -62,6 +62,7 @@ public abstract class CICSOptionsCheckBaseUtility {
           put(CICSParser.RULE_cics_into, "INTO or SET");
         }
       };
+  private boolean noLengthEnabled;
 
   public CICSOptionsCheckBaseUtility(
       DialectProcessingContext context,
```

Comment:
Let this constructor take e.g. `CICSCheckUtilityParameters` with noLengthEnabled boolean.
The  `CICSCheckUtilityParameters` will itself be constructed out of List<String> by
```
for(String opt: ops) {
  switch(opt.toUpperCase()) {
  case "LENGTH": noLengthEnabled = false; break;
  case "NOLENGTH": noLengthEnabled = true; break;
  }
}
```
in the CICSVisitor and passed into CICSOptionsCheckUtility, which will pass it to the individual check utilities.